json/Name_Chip_ActiveName.txt

```diff
@@ -5172,77 +5172,77 @@
 	{
 		"assign": "0",
 		"jp_text": "なし",
-		"tr_text": ""
```

Comment:
Leave these blank

Reply:
as per discussion in discord, checked these and found they were for material chips like star stones